motoman_msgs/srv/ReadMRegister.srv

```diff
@@ -0,0 +1,18 @@
+
+# Read (and return) the current value of the M register at 'address'.
+#
+# Addresses are plain, base-10 integers, as used and displayed by the controller
+# (on the teach pendant for instance).
+#
+# Only the following addresses can be read from:
+#
+#  - 0 to 999
+#
+# Refer also the Yaskawa Motoman documentation on IO addressing and
+# configuration.
+
+uint32 address
```

Comment:
@gavanderhoorn would it make more sense to instead make this a signed int16, techcnially you can't put anything larger than a 16bit number in these m registers. This would also then allow the user to put a negative number in that register as well.

Reply:
Yaskawa controller M-Registers are unsigned 16bit numbers.  Registers are mainly used for timers and counters, so you cannot enter negative values.

Reply:
We could potentially switch to `uint16`s here in the `.srv`.

The main reason why there are `uint32`s used right now is because the MotoROS SimpleMessage structure specs those:

https://github.com/ros-industrial/motoman/blob/7c648e84f623d2c9f8cd640a052efe6ee9c73110/motoman_driver/MotoPlus/SimpleMessage.h#L365-L369

Similar to #436, we could use a narrower type in the `.srv` to prevent users from trying to use values `> 2^16` and then discovering that doesn't work.